test/StatsTestCases.test.js

```diff
@@ -131,7 +131,7 @@ describe("StatsTestCases", () => {
 				actual = actual
 					.replace(/\r\n?/g, "\n")
 					.replace(/[\t ]*Version:.+\n/g, "")
-					.replace(path.join(base, testName), "Xdir/" + testName)
+					.replace(new RegExp(path.join(base, testName), "g"), "Xdir/" + testName)
```

Comment:
This doesn't work for windows paths and paths containing chars that need to be excaped.